cpp-package/include/mxnet-cpp/optimizer.hpp

```diff
@@ -94,15 +94,15 @@ inline unsigned Optimizer::UpdateCount_(int index) {
   return new_count;
 }
 
-inline float Optimizer::GetLR_(int index) {
+inline std::string Optimizer::GetLR_(int index) {
```

Comment:
i meant.. return `stof(params_["lr"])` instead of `lr=stof(params_["lr"])`, so that when `lrScheduler_ != nullptr`, `stof(params_["lr"])` won't be evaluated.